src/JMS/Serializer/Annotation/RecursionGroups.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * Copyright 2013 Andreas Ferber <af+git@chaos-agency.de>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace JMS\Serializer\Annotation;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY"})
+ */
+final class RecursionGroups
```

Comment:
Only single note I have that we can pass associative array instead of ifGroups/withGroups.
We've already discussed it yesterday.